src/main/java/seedu/address/logic/commands/find/FindCommand.java

```diff
@@ -16,25 +13,17 @@
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all items whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "Parameters: KEYWORD [MORE_KEYWORDS]... i/ TYPE\n"
```

Comment:
```suggestion
            + "Parameters: KEYWORD [MORE_KEYWORDS]... " + PREFIX_ITEM + " TYPE\n"
```
Just to be consistent with your example below it!